#rslidar的数据类型0：XYZI，2：XYZIRT
input_type: 2
lidar_topic: "/rslidar_points"
camera_topic_list: ["/camera_front_left/depth/color/points","/camera_front_right/depth/color/points","/camera_back/depth/color/points"]
#融合的相机个数
camera_fusion_quantity: 3
#融合的相机编号（0：前左，1：前右，2：后）
camera_fusion_index: [0,1,2]
#小狗-相机深度点云转换到激光雷达坐标系的转换矩阵的转置
#camera_T0: [0.4226184,0,0.9063077999999999953,0,-0.30997549999999998226,-0.9396926,0.144544,0,0.85165069999999995166,-0.34202019999999996824,-0.3971314,0,0.0988199999999999812,0.045970000000000029184,-0.10292999999999998608,1]
#camera_T1: [0.4226184,0,0.9063077999999999953,0,0.30997549999999998226,-0.9396926,-0.144544,0,0.85165069999999995166,0.34202019999999996824,-0.3971314,0,0.0988199999999999812,-0.045970000000000029184,-0.10292999999999998608,1]
#camera_T2: [1,0,0,0,0,1,0,0,0,0,1,0,0,0,0,1]
#大狗-相机深度点云转换到激光雷达坐标系的转换矩阵的转置
#camera_T: [[-0.4226184, -0.00000000000000002498, -0.9063077999999999953, 0, 0.30997549999999998226, 0.9396926, -0.144544, 0, 0.85165069999999995166, -0.34202019999999996824, -0.3971314, 0, 0.1133, 0.045980000000000031021, -0.10077000000000000628, 1],[-0.4226184, 0.00000000000000002498, -0.9063077999999999953, 0, -0.30997549999999998226, 0.9396926, 0.144544, 0, 0.85165069999999995166, 0.34202019999999996824, -0.3971314, 0, 0.1133, -0.045980000000000031021, -0.10077000000000000628, 1],[0, 1, 0, 0, 0.4999998999999998811, 0, -0.8660253999999998646, 0, -0.86602539999999986459, 0, -0.49999989999999988125, 0, -0.71329999999999983517, -0.017, -0.13664999999999987081, 1]]
camera_T0: [-0.4226184, -0.00000000000000002498, -0.9063077999999999953, 0, 0.30997549999999998226, 0.9396926, -0.144544, 0, 0.85165069999999995166, -0.34202019999999996824, -0.3971314, 0, 0.1133, 0.045980000000000031021, -0.10077000000000000628, 1]
camera_T1: [-0.4226184, 0.00000000000000002498, -0.9063077999999999953, 0, -0.30997549999999998226, 0.9396926, 0.144544, 0, 0.85165069999999995166, 0.34202019999999996824, -0.3971314, 0, 0.1133, -0.045980000000000031021, -0.10077000000000000628, 1]
camera_T2: [0, 1, 0, 0, 0.4999998999999998811, 0, -0.8660253999999998646, 0, -0.86602539999999986459, 0, -0.49999989999999988125, 0, -0.71329999999999983517, -0.017, -0.13664999999999987081, 1]
#点云滤波的深度范围
depth_filter: 1.8

#发布的话题名和坐标系id
pub_topic: "/velodyne_points"
pub_frame_id: "velodyne"
